### ISSUE TITLE
PHP version dependency on composer

### ISSUE THREAD

**Issue by VitorVRS** [PHPOffice/PHPWord#1026]
Hi,
I've noticed your composer.json especify PHP >= 5.3.3 as dependency, but another dependency (zendframework/zend-stdlib:2.4.* for instance) requires PHP 5.3.23 as dependency...
Unfortunately i'm using PHP 5.3.10 and could not run composer install because of that.

I suggest you to change PHP version value (if you'll maintain this compatibility).

Thanks for your great project.

**Comment by troosan** [comment]
@RomanSyroeshko Was there a particular reason to have a dependency to 2.4.*
You changed it from 2.5.* -> 2.4.*, as far as I could see it should not be a problem to switch to 2.2.*
2.2.10 being the last version of zendframework/zend-stdlib to support php 5.3.3, the next version 2.3 requires php 5.3.23

**Comment by RomanSyroeshko** [comment]
Hello @troosan,
Sure. ZF starting from v2.5 supports PHP 5.5 and above. PHPWord still support PHP 5.3. There was an [issue](https://github.com/zendframework/zend-stdlib/issues/28) with `stdlib` dependencies. So, I switched back when found out that problem.

As for PHPWord. I prefer to say "it supports the latest version of PHP 5.3". I wouldn't say that PHPWord completely supports all minor versions starting from PHP 5.3.3. I faced with some issues building PHPWord on that version, so I had to remove it from Travis CI build configuration. Builds were failing from time to time on 5.3.3. So, I would not recommend downgrading third party libraries just to declare full compliance with 5.3.3.
